client/src/plugins/update-checks/NewVersionInfoView.less

```diff
@@ -0,0 +1,55 @@
+:local(.NewVersionInfo) {
+  user-select: none;
+
+  h2 {
+    font-weight: normal;
+  }
+
+  font-size: 1.2em;
+
+  button + button {
+    margin-left: 10px;
+  }
+
+  button {
+    padding: 6px 10px;
+  }
+
+  button:focus {
+    outline: 0 !important;
```

Comment:
Why do we need to remove the outline?

Reply:
I removed it as a personal choice but it makes sense to keep it as we now also have focus in Privacy Settings modal.